build.sbt

```diff
@@ -71,6 +74,24 @@ lazy val browserTestSettings = Seq(
   }
 )
 
+// start the test server before running tests
+def testServerSettings(config: Configuration) = Seq(
+  test in config := (test in config)
```

Comment:
for anybody reading the code, I think it might be valuable information (as a comment) to know why the JS project needs special handling - and cannot be tested like the other backends

Reply:
Done